python/ptest/cases/gomspace_checkout_case.py

```diff
@@ -0,0 +1,104 @@
+# Gomspace test case. Gets cycle count purely for diagnostic purposes and logs
+# any other Gomspace state fields.
+from .base import SingleSatOnlyCase
+
+
+class GomspaceCheckoutCase(SingleSatOnlyCase):
+
+    def setup_case_singlesat(self, simulation):
+        simulation.flight_controller.write_state(
+            "pan.state", 9)  # Manual state
+        self.run_case_singlesat(simulation)
+        print("Gomspace cases finished.")
+
+    def run_case_singlesat(self, simulation):
+        simulation.cycle_no = simulation.flight_controller.read_state(
+            "pan.cycle_no")
+
+        # readable fields
+
+        vboost = [
+            simulation.flight_controller.read_state("gomspace.vboost.output1"),
+            simulation.flight_controller.read_state("gomspace.vboost.output2"),
+            simulation.flight_controller.read_state("gomspace.vboost.output3")]
+
+        vbatt = simulation.flight_controller.read_state("gomspace.vbatt")
+        if vbatt < 6.0 or vbatt > 8.4:
+            print("Vbatt is out of expected range [6, 8.4] at: " + vbatt)
+
+        currin = [
+            simulation.flight_controller.read_state("gomspace.curin.output1"),
+            simulation.flight_controller.read_state("gomspace.curin.output2"),
+            simulation.flight_controller.read_state("gomspace.curin.output3")]
```

Comment:
You can probably write a list comprehension on the state field name to shorten the code here? Same with the other repetitive state fields below.